src/PowerShellEditorServices/Session/PowerShellContext.cs

```diff
@@ -174,6 +191,13 @@ public static Runspace CreateRunspace(PSHost psHost)
             }
 
             Runspace runspace = RunspaceFactory.CreateRunspace(psHost, initialSessionState);
+
+            // Windows PowerShell must be hosted in STA mode
+            if (RuntimeInformation.FrameworkDescription == DotNetFrameworkDescription)
```

Comment:
Ends up being the same thing I believe, but I usually write this as

```csharp
if (RuntimeInformation.FrameworkDescription.Equals(DotNetFrameworkDescription, StringComparison.Ordinal))
```

If we ever start using code factor or some other static analyzer that often gets flagged.